README.md

```diff
@@ -32,66 +32,91 @@ A tutorial for manim, a mathematical animation engine made by 3b1b for Python.
 * [Resources](#Resources)
 * [Further Work](#Further-Work)
 * [Acknowledgements](#Acknowledgements)
+
 ## Installations
-This installation and guide is meant for linux users. We will start by installing prerequisites. 
 
-Lets first get the repo for manim, using:
-``` bash
-    git clone https://github.com/3b1b/manim.git
-```
+Assuming you have Python 3 installed and set up properly on your machine.
+
+Since [a commit][commit] by [@JohnAZoidberg][committer] in April, manim has been made installable using Python's tool `pip`, so the installation process is much simpler than before.
 
+[commit]: https://github.com/3b1b/manim/commit/2a6918cc30336544856a68f065a21ba2b769b01c
+[committer]: https://github.com/JohnAZoidberg
+
+### Linux
 We will start with installing some system requirements: Cairo, Latex, ffmpeg and sox. 
 
-Install by running the following commands on the terminal: 
+```shell
+$ sudo apt-get install libcairo2-dev libjpeg-dev libgif-dev 
+$ sudo apt-get install texlive-latex-base texlive-full texlive-fonts-extra
+```
 
-``` bash
-    sudo apt install SystemReq
+After having these libraries and tools installed in your computer, you can now type this command to finish the last step of the installation
+```shell
+$ pip3 install manim
 ```
+> Don't worry, `pip` automatically gathers Python dependencies of manim
 
-Some additional installations are mentioned below
-``` bash
-    sudo apt-get install libcairo2-dev libjpeg-dev libgif-dev 
-    sudo apt install texlive-latex-base texlive-full texlive-fonts-extra
+### Mac OSX
+A popular package manager called [HomeBrew][brew_website] is recommended for coding on a Mac because you do not have to go to App Store to or download dmg files from websites.
+
+To support rendering SVG files when creating videos, you need to install MacTex either by downloading from [its website][mactex_website] or using HomeBrew:
+```shell
+$ brew cask install mactex
 ```
-Phew! This will be the last installation for additional python modules. Run this in the terminal 
-``` bash
-    python3 -m pip install -r requirements.txt
+
+At last, use `pip` to install your manim:
+```shell
+$ pip3 install manim
 ```
 
-### Common-Problems
-* Problem #1: Cairo System requirement 
-People are sometimes unable to install cairo through the terminal. But, it is possible to install it using the Python.
-``` bash
-        pip3 install pycairo
+[brew_website]: https://brew.sh
+[mactex_website]: https://www.tug.org/mactex/
+
+### Common Problems
+1. Cairo System requirement 
+
+People are sometimes unable to install cairo through the terminal, but luckily Python can help us fix this:
+```shell
+$ pip3 install pycairo
 ```
 
-* Problem #2: **Exception: Latex error converting to dvi. See log output above or the log file**
```

Comment:
Keep original. I will take all your syntax changes into account, and give you credit for it.